ignitionrobotics/ign-rendering
# Add SetSize API for LidarVisual

Signed-off-by: Ian Chen <ichen@osrfoundation.org>

# 🎉 New feature

## Summary

Support setting point size for lidar visual. 

~Currently working for ogre 1.x only. I still need to figure out how to make it work with the ogre2 hlms system.~

**Update**: ogre2 is now working - updated to use ogre low level materials.

## Test it

1. Build and run the [lidar_visual](https://github.com/ignitionrobotics/ign-rendering/tree/lidar_point_size/examples/lidar_visual) example. 
2. Press `2` to toggle to point rendering and see the points are now slightly larger than before (size increased from 1 to 5):

![lidar_visual_size_5](https://user-images.githubusercontent.com/4000684/131755575-e7f9023b-b746-4145-8c37-fad36d65075c.png)


## Checklist

**Note to maintainers**: Remember to use **Squash-Merge**
